statsmodels/tsa/statespace/_initialization.pxd

```diff
@@ -0,0 +1,111 @@
+#cython: boundscheck=False
+#cython: wraparound=False
+#cython: cdivision=False
+"""
```

Comment:
Does everything here need to be exposed externally?

Reply:
I missed responding to this earlier, but I'm not sure what you mean by this - that the classes should be prefixed by underscores?

Reply:
One of the purposes of the pxd file is to make it expose things to be `cimport`ed.  (The other big one is for type declarations, but since you already have those in the pyx file that is unnecessary).  So anything that is not needed outside of statespace._intialization can be excluded from the pxd file.

The main benefit of this is for the reviewer: if it is easy to check that a function/class is only used internally, it becomes much easier to track reason about its use cases.